docker/test/stress/stress

```diff
@@ -146,6 +146,12 @@ def prepare_for_hung_check(drop_databases):
             "KILL QUERY WHERE query LIKE 'SELECT URL, uniq(SearchPhrase) AS u FROM test.hits GROUP BY URL ORDER BY u %'"
         )
     )
+    # Long query from 02136_kill_scalar_queries
+    call_with_retry(
+        make_query_command(
+            "KILL QUERY WHERE query LIKE 'SELECT (SELECT number FROM system.numbers WHERE number = 1000000000000)%'"
```

Comment:
LGTM, but why not `KILL QUERY WHERE 1`?

Reply:
It will not find issues like https://github.com/ClickHouse/ClickHouse/issues/21004 with `WHERE 1`. But I'm not sure if we really want to find them...